package.json

```diff
@@ -51,7 +51,7 @@
     "vx"
   ],
   "license": "Apache-2.0",
-  "devDependencies": {
+  "dependencies": {
```

Comment:
The root package.json doesn't have a production release anyway. This change is needed to appease linting in `scripts`.